spec/checker_jobs/dsl_spec.rb

```diff
@@ -3,11 +3,16 @@
 
   let(:key) { Object.new }
   let(:block) { Proc.new {} }
-  let(:target) { Object.new }
+  let(:notifier) { :action_mailer }
+  let(:notifier_options) { { to: "tech@drivy.com", from: "checkers@drivy.com" } }
   let(:default) { Object.new }
   let(:options) { Object.new }
   let(:duration) { Object.new }
 
+  before :all do
```

Comment:
RSpec/BeforeAfterAll: Beware of using `before :all` as it may cause state to leak between tests. If you are using `rspec-rails`, and `use_transactional_fixtures` is enabled, then records created in `before :all` are not automatically rolled back.